roles/build-release/tasks/tests.yaml

```diff
@@ -124,7 +124,5 @@
         {{ antsibull_ansible_venv }}/bin/ansible-playbook -i 'localhost,' --connection=local
           -e antsibull_sdist_dir="{{ antsibull_sdist_dir }}"
           -e antsibull_ansible_venv="{{ antsibull_ansible_venv }}"
-          -e antsibull_ansible_git_dir="{{ antsibull_ansible_git_dir }}"
-          -e antsibull_ansible_git_version="{{ antsibull_ansible_git_version }}"
```

Comment:
These two variables are no longer used. I'm not sure if you plan to replace this test in a separate PR or if we should remove them from the argument spec. In any case, I'm fine with merging as is to fix our CI.

Reply:
`antsibull_ansible_git_repo` is also no longer used. I removed all three in fbf3b4d1cf4ab987eeafaee602ef7f50f5c90abe.